css/properties/-webkit-transform-origin-y.json

```diff
@@ -20,7 +20,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": "≤13.1"
+              "version_added": "≤5.1"
```

Comment:
❓ This property is not reported by computed properties, so I'm not sure about this one.